src/components/panel/panel.spec.js

```diff
@@ -91,6 +91,52 @@ describe('$mdPanel', function() {
     expect(PANEL_WRAPPER_CLASS).not.toHaveClass(HIDDEN_CLASS);
   });
 
+  it('should trap focus inside of the panel with trapFocus=true', function() {
+    var template = '<div>Hello World!</div>';
```

Comment:
optional: We've now used this template probably a dozen tests. What do you think of moving it to the top of the file as a constant? DEFAULT_TEMPLATE perhaps? It doesn't actually matter for the tests.